source/sass/31-molecules/oidc-menu/_oidc-menu.scss

```diff
@@ -0,0 +1,161 @@
+/* ==========================================================================
+   OIDC Menu block
+   ========================================================================== */
+
+.authentication {
```

Comment:
Is dit specifiek genoeg? Lijkt mij een vrij algemene class. Is er niets met de naam van de module zelf?

Reply:
Heb t aangepast